packages/jest-config/src/resolveConfigPath.ts

```diff
@@ -137,20 +137,18 @@ function extraIfPackageJson(configPath: string) {
   return '';
 }
 
-const makeMultipleConfigsWarning = (configPaths: Array<string>) =>
-  chalk.yellow(
-    [
-      chalk.bold('\u25cf Multiple configurations found:'),
-      ...configPaths.map(
-        configPath =>
-          `    * ${extraIfPackageJson(configPath)}${slash(configPath)}`,
-      ),
-      '',
-      '  Implicit config resolution does not allow multiple configuration files.',
-      '  Either remove unused config files or select one explicitly with `--config`.',
-      '',
-      '  Configuration Documentation:',
-      '  https://jestjs.io/docs/configuration.html',
-      '',
-    ].join('\n'),
-  );
+const makeMultipleConfigsErrorMessage = (configPaths: Array<string>) =>
+  [
+    chalk.bold('\u25cf Multiple configurations found:'),
```

Comment:
```suggestion
    chalk.bold(`${BULLET} Multiple configurations found:`),
```

`BULLET` is in `./utils`

Reply:
okay, done.